components/encryption/src/manager/mod.rs

```diff
@@ -177,12 +177,13 @@ impl Dicts {
     }
 
     fn delete_file(&mut self, fname: &str) -> Result<()> {
-        let file = self.file_dict.files.remove(fname).ok_or_else(|| {
-            Error::Io(IoError::new(
-                ErrorKind::NotFound,
-                format!("file not found, {}", fname),
-            ))
-        })?;
+        let file = match self.file_dict.files.remove(fname) {
+            Some(file_info) => file_info,
+            None => {
+                // Could be a plaintext file not tracked by file dictionary.
+                return Ok(());
```

Comment:
Instead of returning Ok, how about handle NotFound error at a higher level? I worry about we may miss some error somewhere.

Reply:
Discussed offline, will add more info log instead.